backend/db/ddl-16-concept_relationship_plus.jinja.sql

```diff
@@ -6,58 +6,35 @@
 -- for now, due to bug (https://github.com/jhu-bids/TermHub/issues/191 and https://github.com/jhu-bids/TermHub/pull/190)
 -- filtering out cr records including invalid concepts. this is probably not the right thing to do
 -- in the long term, but should fix bug and let us move forward with immediate need to get pilot started (2022-01-4)
-DROP TABLE IF EXISTS {{schema}}concept_relationship_plus CASCADE;
+DROP TABLE IF EXISTS {{schema}}concept_relationship_plus{{optional_suffix}} CASCADE;
 
-CREATE TABLE IF NOT EXISTS {{schema}}concept_relationship_plus AS (
+CREATE TABLE IF NOT EXISTS {{schema}}concept_relationship_plus{{optional_suffix}} AS (
   SELECT    c1.vocabulary_id AS vocabulary_id_1
           , cr.concept_id_1
           , c1.concept_name AS concept_name_1
           , c1.concept_code
-          , c1.total_cnt AS total_cnt_1,
```

Comment:
@Sigfried Bugfix; was an extra `,`
- Added some missing `{{optional_suffix}}` to `concept_relationship_plus` DDL

Reply:
Bugfix: Reran and there was another extra `,`

Reply:
Bugfix: Re-ran, encountered, and fixed:
`sqlalchemy.exc.ProgrammingError: (psycopg2.errors.DuplicateColumn) column "total_cnt_1" specified more than once`